privacy.html

```diff
@@ -39,7 +41,10 @@ <h3>Do we use 'cookies'?</h3>
             <p>If users disable cookies in their browser:</p>
             <p>If you turn cookies off, Some of the features that make your site experience more efficient may not function properly.</p>
             <h3>Third-party disclosure</h3>
-            <p>We do not sell, trade, or otherwise transfer to outside parties your Personally Identifiable Information.</p>
+            <p>We will not share, rent, sell or otherwise disclose any personal information that we collect about you, except in any of the following situations or otherwise described in this Privacy Policy:</p>
+              <ul>
+                <li>We disclose your public IP with a secure service provider that help us determine your location in order to customize our service and provide localized content. We do not authorize them to use or disclose your personal information except in connection with providing their services (which includes maintaining and improving their services).</li>
+              </ul>
             <h3>Third-party links</h3>
```

Comment:
We will not share, rent, sell or otherwise disclose any personal information that we collect about you, except in any of the following situations or otherwise described in this Privacy Policy:

 We disclose your public IP with a secure service provider that helps us determine your location in order to customize our service and provide localized content. We do not authorize them to use or disclose your personal information except in connection with providing their services (which includes maintaining and improving their services).